src/test/bridges/example/ExampleE2E.t.sol

```diff
@@ -15,7 +15,7 @@ import {ErrorLib} from "../../../bridges/base/ErrorLib.sol";
  *         as possible without spinning up all the rollup infrastructure (sequencer, proof generator etc.).
  */
 contract ExampleE2ETest is BridgeTestBase {
-    IERC20 public constant DAI = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
+    IERC20 public constant USDC = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
```

Comment:
Why have you changed the name? The address is DAI.

Reply:
I forgot to change the address. I changed It to USDC because I want grantees to see asset registration in the example tes. I will fix the address.

Reply:
Addressed in 1eb4e516e8330e4c6bf0647f9b6710e217804d0f